pkg/haproxy/instance_test.go

```diff
@@ -4912,7 +4939,13 @@ func TestModSecurity(t *testing.T) {
 		globalModsec.Endpoints = test.endpoints
 		globalModsec.Timeout.Connect = "1s"
 		globalModsec.Timeout.Server = "2s"
-		globalModsec.Args = test.modsecAgentArgs
+		if test.modsecAgentArgs != nil {
+			globalModsec.Args = test.modsecAgentArgs
+		} else {
+			globalModsec.Args = strings.Split(ingtypes.DefaultCorazaArgs, " ")
```

Comment:
You can add arbitrary strings here, it doesn't need to be the real configuration. A simple copy/paste from the real data is fine.